src/Polysemy/Writer.hs

```diff
@@ -122,3 +127,122 @@ runWriterAssocR =
     {-# INLINE go #-}
   in fmap (first ($ mempty)) . go
 {-# INLINE runWriterAssocR #-}
+
+------------------------------------------------------------------------------
+-- | Run a 'Writer' effect by transforming it into atomic
+-- operations over an 'IORef'.
+--
+-- /Note/: All 'tell's in the argument of a 'listen' or 'pass' will temporarily
```

Comment:
Yeah, this is a little weird, but I guess it makes sense.

Reply:
It's borderline for me. Perhaps it's better to wait until `Final` drops before I write `runWriterTVar`? I.e. only keep `runOutputMonoidIORef` and `runOutputMonoidTVar`?